pkg/machine/e2e/init_windows_test.go

```diff
@@ -75,4 +77,50 @@ var _ = Describe("podman machine init - windows only", func() {
 		Expect(session).To(Exit(125))
 		Expect(session.errorToString()).To(ContainSubstring("already exists on hypervisor"))
 	})
+
+	It("init should not should not overwrite existing HyperV vms", func() {
```

Comment:
Tough sentence to parse, or were you going for emphasis on the "should not should not" :^)